source/guide_fittrackee.rst

```diff
@@ -0,0 +1,186 @@
+
+
+Then install version 1.26.6 of ``urllib3`` because otherwise you will get an error message later when setting up the database.
+
+::
+
+ (fittrackee_venv) [isabell@stardust fittrackee]$ pip3.9 install urllib3==1.26.6
+ ...
+ Successfully installed urllib3-1.26.6
+
+
+Configuration
+=============
+
+PostgreSQL
+----------
+
+Please follow the :manual:`PostgreSQL <guide_postgresql>` guide to configurate PostgreSQL.
+
+After that `create fittrackee database  <https://samr1.github.io/FitTrackee/en/installation.html#from-pypi>`_
+
+Web Backends
+------------
+
+With web backends you can connect Fittrackee running on port 5000 by default to the frontend to make it accessible from outside.
+Please follow the instructions :manual:`Web Backends <web-backends>`.
+
+
+Optional: Redis
+---------------
+
+We need Redis for task queue (if email sending is enabled and for data export requests) and API rate limits.
+
+Please follow the :lab:`Redis <guide_redis>` guide to setup redis with one variation: By default, Redis does not run on a port with us, but provides a Unix socket under ``/home/$USER/.redis/sock``. Either one has to adjust the configuration of the Fittrackee application or change the Redis configuration from `port 0' to 6379.
+
+Optional: Emails
+----------------
+
+In order to be able to send e-mails from Fittrackee, you need an e-mail account, which you specify in the next step in the file ``.env``. Any account can be used for this purpose. If you like, you can create the account directly at Uberspace.
+
+Please follow the :manual:`mail <mail-access>` guide to setup up Emails.
+
+
+Environment variables for Fittrackee
+------------------------------------
+
+You need to create the necessary environment variables for Fittrackee.
+You can use the `example file <https://github.com/SamR1/FitTrackee/blob/master/.env.example>`_ as a guide.
+Further explanations can be found in the `Fittrackee documentation <https://samr1.github.io/FitTrackee/en/installation.html#environment-variables>`_.
+
+Special note: For an installation in our system it is important to customize the variables ``HOST`` and ``DATABASE_URL`` as follows:
+
+::
+
+ export HOST=0.0.0.0
+ ...
+ export DATABASE_URL=postgresql://fittrackee:fittrackee@localhost:5432/fittrackee
+ ...
+
+For more information about the available ports, see :manual:`firewall ports <basics-ports>`.
+
+
+Set up and launch Fittrackee
+----------------------------
+
+Start virtual environment:
+
+::
+
+ [isabell@stardust ~]$ cd ~/fittrackee/
+ [isabell@stardust fittrackee]$ source fittrackee_venv/bin/activate
+ (fittrackee_venv) [isabell@stardust fittrackee]$
+
+
+Set environment variables from file ``.env``:
+
+::
+
+ (fittrackee_venv) [isabell@stardust fittrackee]$ source /PathToYourEnvFile/.env
+
+Initialize database schema:
+
+::
+
+ (fittrackee_venv) [isabell@stardust fittrackee]$ ftcli db upgrade
+
+Start the application
+
+::
+
+ (fittrackee_venv) [isabell@stardust fittrackee]$ fittrackee
+
```

Comment:
What happens when we reboot the server? Can you please provide a supervisord service that monitores and keeps your service running? Check out [the manual](https://manual.uberspace.de/daemons-supervisord/). We reboot our servers on a regular basis and sometimes they crash and have to be rebooted to get everything back up running. In your version of the guide every user needs to login after every reboot and start `fittrackee` by hand.